lib/jest-helpers.js

```diff
@@ -5,7 +5,7 @@ const cheerio = require('cheerio')
 const { configureAxe } = require('jest-axe')
 const sass = require('node-sass')
 const nunjucks = require('nunjucks')
-const outdent = require('outdent')
+const { outdent } = require('outdent')
```

Comment:
Likely another ESM to CJS transpiler "synthetic default" issue. Worked fine before, but the compiler has flagged that we should be using the named export (as discussed in: https://github.com/alphagov/govuk-frontend/issues/2829)